jni/Android-app.mk

```diff
@@ -0,0 +1,382 @@
+LOCAL_PATH := $(call my-dir)
+#
+# Geunsik Lim <geunsik.lim@samsung.com>
+# This configuration file is to compile a test application
+# using Gstreamer + NNstreamer library.
+#
+# Step1: Build a test appliation based on nnstreamer for Android platform
+# ndk-build APP_BUILD_SCRIPT=./Android-app.mk
+#
+# Step2: Install a test application into Android target device
+# readelf -d ../libs/arm64-v8a/{your-test-app}
+# adb push   ../libs/arm64-v8a/{your-test-app} /data/nnstreamer/
+#
+# Step3: Run the test application
+# adb shell
+# target#> cd /data/nnstreamer/
+# target#> ./{your-test-app}
+
+NNSTREAMER_VERSION := 0.1.1
+TARGET_ARCH_ABI    := arm64-v8a
+
+ifeq ($(TARGET_ARCH_ABI),armeabi)
+GSTREAMER_ROOT        := $(GSTREAMER_ROOT_ANDROID)/arm
+else ifeq ($(TARGET_ARCH_ABI),armeabi-v7a)
+GSTREAMER_ROOT        := $(GSTREAMER_ROOT_ANDROID)/armv7
+else ifeq ($(TARGET_ARCH_ABI),arm64-v8a)
+GSTREAMER_ROOT        := $(GSTREAMER_ROOT_ANDROID)/arm64
+else ifeq ($(TARGET_ARCH_ABI),x86)
+GSTREAMER_ROOT        := $(GSTREAMER_ROOT_ANDROID)/x86
+else ifeq ($(TARGET_ARCH_ABI),x86_64)
+GSTREAMER_ROOT        := $(GSTREAMER_ROOT_ANDROID)/x86_64
+else
+$(error Target arch ABI not supported: $(TARGET_ARCH_ABI))
+endif
+
+# Describe shared libraries that are needed to run this application.
+include $(CLEAR_VARS)
+LOCAL_MODULE := gstreamer-1.0
+LOCAL_SRC_FILES := $(GSTREAMER_ROOT)/lib/libgstreamer-1.0.so
+include $(PREBUILT_SHARED_LIBRARY)
+
```

Comment:
How about using foreach and define to reduce code length?

Reply:
Thanks. PR will be updated.